cryptoinvestor/templates/assets/assets.html

```diff
@@ -19,6 +27,14 @@
                     <td>{{ asset.id }}</td>
                     <td>{{ asset.name }}</td>
                     <td>{{ asset.rate }}</td>
+                    <td> 
+                        <input id="{{asset.name}}" type="number" data-length="6">
```

Comment:
Asset's id should be enough to uniquely identify it.

Reply:
Also we should probably restrict the the input to only take valid values (no negative numbers for example). And also write tests for the underlying function that is being called by this.